blueocean-core-js/src/js/paths/rest.js

```diff
@@ -12,12 +12,11 @@ export default {
         return '/blue/rest';
     },
 
-    organizationPipelines(organizationName) {
-        return `${this.apiRoot()}/search/?q=type:pipeline;organization:${encodeURIComponent(organizationName)};excludedFromFlattening:jenkins.branch.MultiBranchProject,hudson.matrix.MatrixProject&filter=no-folders`;
-    },
+    pipelines(organizationName, searchText) {
```

Comment:
hrm... this isn't really a very practical "first time" PR - (to @imeredith) - does the pipeline/page stuff really need to be in core-js? There doesn't seem to be another other modules using it (eg only dashboard changed). Is this premature abstraction?